pkg/nuclide/server/scripts/utils.py

```diff
@@ -46,7 +46,7 @@ def http_get(host, port, method, url, key_file=None, cert_file=None, timeout=1):
             return ret
         else:
             return None
-    except:
+    except Exception:
```

Comment:
I meant to say that that is a good thing. I can't think of why you would intentionally want to capture system exits and/or keyboard interrupts. I think we should merge this.